docs/pages/reference/metrics.mdx

```diff
@@ -2,12 +2,38 @@
 title: Teleport Metrics
 description: Comprehensive list of all metrics exposed by Teleport.
 ---
-The following metrics are available:
 
 <Notice scope={["cloud"]} type="tip">
 
 Teleport Cloud does not expose monitoring endpoints for the Auth Service and Proxy Service.
 
 </Notice>
 
+Teleport metrics data is primarily used for performance monitoring. If you'd like to
+monitor Teleport usage, consider utilizing our *fluentd* plugin to push Audit Events into your
+preferred logging aggregation system (Elastic, Splunk, Sumo, etc). 
+
+<ScopedBlock scope="oss">
```

Comment:
These three `ScopedBlock`s are the same for all scopes. Should we collapse them into one bulleted list without the `ScopedBlock` tag?

Reply:
New to this... I was trying to align with the implementations I saw on other pages. It seems the ScopedBlock adds the appropriate URL query string, depending on the current scope. i.e. If I'm on OSS, clicking on the enclosed links would keep me on the OSS scoped docs. Let me test this out a bit more.

Reply:
Ah! The URL query params follow you regardless. Cool, I can remove this.